### DISCUSSION TITLE
Anyone using shadcn/ui with react_on_rails ?

### DISCUSSION THREAD

**Discussion by chrisedington** [shakacode/react_on_rails#1661] [Q&A]
Anyone managed to get shadcn/ui working within a React on Rails context? 

Would love any info anyone has, it's not as straight forward as I imagined it would be for something like MUI / Ant Design.

**Comment by justin808** [comment]
@chrisedington I've never tried to use https://ui.shadcn.com/. 

We've done tons with Tailwind.

My team and I are pretty good at figuring out issues. Let us know if we can help you. 

Check out https://www.shakacode.com/blog/client-engagement-model/#1-technical-optimizations--developer-environments.

You can email me at justin@shakacode.com.
